collector/pg_process_idle.go

```diff
@@ -15,21 +15,23 @@ package collector
 
 import (
 	"context"
+	"database/sql"
 
 	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const processIdleSubsystem = "process_idle"
-
 func init() {
-	registerCollector(processIdleSubsystem, defaultEnabled, NewPGProcessIdleCollector)
+	// Making this default disabled because we have no tests for it
+	registerCollector("statements", defaultDisabled, NewPGProcessIdleCollector)
```

Comment:
Why was this renamed?

Reply:
Good question. Probably a typo somewhere